get_genotype_vcf.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+# pylint: disable=import-error
+
+"""
+This script will
+
+- extract common variants
+- export as VCF (one per chrom)
+
+this will be used as input for the
+SAIGE-QTL association pipeline.
+"""
+
+# python modules
+from cpg_utils.hail_batch import (
+    dataset_path,
+    get_batch,
+    get_config,
+    init_batch,
+    output_path,
+)
+
+import click
+import hail as hl
+from hail.methods import export_vcf
+
+
+BIOHEART_TOB_VDS = dataset_path('vds/1-0.vds')
+BCFTOOLS_IMAGE = get_config()['images']['bcftools']
+
+# inputs:
+@click.option('--chromosomes', help=' eg chr22')
+@click.command()
+def main(chromosomes):
+    """
+    Run associaTR processing pipeline
+    """
+
+    init_batch()
+
+    vds_path = BIOHEART_TOB_VDS
```

Comment:
do you want this to be configurable at any point? NW if this is a one-off script

Reply:
mm i did think that in some ways this is similar to the demux script and maybe i could make it more general? I might end up running this many times, but always on the BioHEART TOB joint call object i believe

Reply:
On the basis that the joint-calls will change name each time, I think providing a name as a command line arg would be good. You don't want a PR to change each input